ports/nrf/boards/sparkfun_nrf52840_micromod/pins.c

```diff
@@ -0,0 +1,112 @@
+    { MP_ROM_QSTR(MP_QSTR_NFC1), MP_ROM_PTR(&pin_P0_09) },      //              NFC1 alias
+    { MP_ROM_QSTR(MP_QSTR_ADC_DM), MP_ROM_PTR(&pin_P0_09) },    //              ADC_DM alias
+    { MP_ROM_QSTR(MP_QSTR_CAM_HSYNC), MP_ROM_PTR(&pin_P0_09) }, //              CAM_HSYNC alias
+    { MP_ROM_QSTR(MP_QSTR_G10), MP_ROM_PTR(&pin_P0_10) },       // 0.10 - G10 | GPIO10/NFC2 | ADC_D+ | CAM_VSYNC (NFC2)
+    { MP_ROM_QSTR(MP_QSTR_NFC2), MP_ROM_PTR(&pin_P0_10) },      //              NFC2 alias
+    { MP_ROM_QSTR(MP_QSTR_ADC_DP), MP_ROM_PTR(&pin_P0_10) },    //              ADC_DP alias
+    { MP_ROM_QSTR(MP_QSTR_CAM_VSYNC), MP_ROM_PTR(&pin_P0_10) }, //              CAM_VSYNC alias
+                                                                // NC   - G11
+
+    // PWM pins (PWM0-PWM1)
+    { MP_ROM_QSTR(MP_QSTR_PWM0), MP_ROM_PTR(&pin_P0_06) },      // 0.06 - PWM0
+    { MP_ROM_QSTR(MP_QSTR_PWM1), MP_ROM_PTR(&pin_P0_16) },      // 0.16 - PWM1
+
+    // PDM
+    { MP_ROM_QSTR(MP_QSTR_PDM_CLK), MP_ROM_PTR(&pin_P0_25) },   // 0.25 - PDM_CLK | AUD_BCLK
+    { MP_ROM_QSTR(MP_QSTR_PDM_DATA), MP_ROM_PTR(&pin_P0_26) },  // 0.26 - PDM_DATA | AUD_LRCLK
+
+    // Battery Voltage Monitor
+    { MP_ROM_QSTR(MP_QSTR_BATT_VIN3), MP_ROM_PTR(&pin_P0_30) }, // 0.30 - BATT_VIN/3 (AIN6)
+
+    // I2C
+    { MP_ROM_QSTR(MP_QSTR_SDA), MP_ROM_PTR(&pin_P0_08) },       // 0.08 - SDA
+    { MP_ROM_QSTR(MP_QSTR_SCL), MP_ROM_PTR(&pin_P0_11) },       // 0.11 - SCL (TRACEDATA2)
+
+    { MP_ROM_QSTR(MP_QSTR_I2C_INT), MP_ROM_PTR(&pin_P0_15) },   // 0.15 - I2C_INT
+
+    { MP_ROM_QSTR(MP_QSTR_SDA1), MP_ROM_PTR(&pin_P1_01) },       // 1.01 - SDA1
+    { MP_ROM_QSTR(MP_QSTR_SCL1), MP_ROM_PTR(&pin_P0_24) },       // 0.24 - SCL1
+
+    // SPI
+    { MP_ROM_QSTR(MP_QSTR_CIPO), MP_ROM_PTR(&pin_P0_02) },      // 0.02 - CIPO | SPI_CIPO
+    { MP_ROM_QSTR(MP_QSTR_MISO), MP_ROM_PTR(&pin_P0_02) },      //              MISO alias
+    { MP_ROM_QSTR(MP_QSTR_COPI), MP_ROM_PTR(&pin_P0_31) },      // 0.31 - COPI | SPI_COPI (AIN7)
+    { MP_ROM_QSTR(MP_QSTR_MOSI), MP_ROM_PTR(&pin_P0_31) },      //              MOSI alias
+    { MP_ROM_QSTR(MP_QSTR_SCK), MP_ROM_PTR(&pin_P0_28) },       // 0.28 - SCK  | SPI_SCK (AIN4)
+    { MP_ROM_QSTR(MP_QSTR_CS), MP_ROM_PTR(&pin_P0_20) },        // 0.20 - /CS  | SPI_/CS
+
+    // Flash SPI
+    { MP_ROM_QSTR(MP_QSTR_IOSCK), MP_ROM_PTR(&pin_P0_19) },     // 0.00 - IOSCK | Flash Serial Clock
+    { MP_ROM_QSTR(MP_QSTR_IOCSN), MP_ROM_PTR(&pin_P0_12) },     // 0.00 - IOCSN | Flash /Chip Select
+    { MP_ROM_QSTR(MP_QSTR_IO0), MP_ROM_PTR(&pin_P0_14) },       // 0.00 - IO0 | Flash Data 0
+    { MP_ROM_QSTR(MP_QSTR_IO1), MP_ROM_PTR(&pin_P0_21) },       // 0.00 - IO1 | Flash Data 1
+    { MP_ROM_QSTR(MP_QSTR_IO2), MP_ROM_PTR(&pin_P0_23) },       // 0.00 - IO2 | Flash Data 2
+    { MP_ROM_QSTR(MP_QSTR_IO3), MP_ROM_PTR(&pin_P1_00) },       // 0.00 - IO3 | Flash Data 3
```

Comment:
We don't usually have this in board because it is used internally. Since it is broken out on the connector, I would include it with the connector naming.